vassal-app/src/main/java/VASSAL/build/GameModule.java

```diff
@@ -316,13 +318,29 @@ public boolean isTrueMovedSupport() {
   }
 
   /**
-   * @param fancyMoveSupport true if a ignore-small-moves trait exists in the module
+   * @param trueMoveSupport true if a ignore-small-moves trait exists in the module
```

Comment:
Param name doc doesn't match code.

Reply:
I'm absolutely baffled that you're seeing this here, because it's not in my code that I pushed, and I even went back to master & then re-checked it out just in case to make sure ?!?!

Reply:
Oh I see it was a 1-letter-off thing. Fixed.